iib/workers/tasks/build.py

```diff
@@ -916,6 +931,8 @@ def handle_add_request(
     :param str overwrite_from_index_token: the token used for overwriting the input
         ``from_index`` image. This is required for non-privileged users to use
         ``overwrite_from_index``. The format of the token must be in the format "user:password".
+    :param str distribution_scope: the scope for distribution of the index image, defaults to
+        'prod'.
```

Comment:
This doesn't default to `prod` as far as I'm aware based on the keyword argument default of `None` and `_validate_distribution_scope` just returns the resolved scope. If there is no resolved scope, then it defaults to `prod`.